svg/attributes/presentation.json

```diff
@@ -3079,40 +3079,40 @@
             "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Attribute/word-spacing",
             "support": {
               "chrome": {
-                "version_added": null
+                "version_added": "1"
               },
               "chrome_android": {
-                "version_added": null
+                "version_added": "18"
               },
               "edge": {
-                "version_added": null
+                "version_added": "12"
               },
               "firefox": {
-                "version_added": null
+                "version_added": "72"
               },
               "firefox_android": {
-                "version_added": null
+                "version_added": false
               },
               "ie": {
-                "version_added": null
+                "version_added": "9"
               },
               "opera": {
-                "version_added": null
+                "version_added": true
               },
               "opera_android": {
-                "version_added": null
+                "version_added": true
               },
               "safari": {
-                "version_added": null
+                "version_added": "5.1"
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": "5.1"
               },
               "samsunginternet_android": {
-                "version_added": null
+                "version_added": "1.0"
               },
               "webview_android": {
-                "version_added": null
+                "version_added": "37"
```

Comment:
Everything else here looks good to me, but I wanted to check on this one as it stood out. Was this supposed to be ≤37 like the other WebView updates here? Or were you able to find this one specifically? Thanks!

Reply:
thanks for the catch - I just missed it on this one. Updated.